Dockerfile

```diff
@@ -1,16 +1,18 @@
 # Use the official Python 3.9 slim image as the base image
-FROM python:3.9-slim as builder
+FROM cgr.dev/chainguard/python:latest-dev as builder
```

Comment:
change base image and only copy whats needed to 
cgr.dev/chainguard/python:latest